### DISCUSSION TITLE
Can a single rule chain multiple conditions in its when condition?

### DISCUSSION THREAD

**Discussion by nagarajtrivedi** [hyperjumptech/grule-rule-engine#222] [Q&A] [answered]
Hi all, can the different values of the same data be verified through a single rule. 
For example
I have a data D1 which can have values value1, value2, value3 and value4
Can this be verified through a single rule OR
Do we need to write 4 different rules

Please clarify

Thanks
Nagaraj

**Comment by niallnsec** [comment, marked as answer]
You can use the local OR operator `||` to validate multiple different conditions in the when condition of a single rule. For example:
```
when D1 == "value1" || D1 == "value2" || D1 == "value3" ....
```

**Reply by nagarajtrivedi** [reply]
Thanks for your early reply

Thanks
Nagaraj
